staging/src/k8s.io/apiserver/pkg/storage/value/encrypt/aes/aes_test.go

```diff
@@ -193,8 +193,8 @@ func benchmarkGCMRead(b *testing.B, keyLength int, valueLength int, stale bool)
 		if err != nil {
 			b.Fatal(err)
 		}
-		if stale {
```

Comment:
Here and below there was a bug: local `stale` variable was shadowing outer variable. In fact it was testing that data could not be stale and tests worked because we didn't reverse the keys.

Reply:
Good catch